src/index.css

```diff
@@ -40,6 +40,11 @@ code {
   left: 50%;
 }
 
+.smooth-disabled {
+  transition: opacity 2s cubic-bezier(0.55, 0.06, 0.82, 0.18),
```

Comment:
We should only have slow animation for disconnecting players. Connecting ones shouldn't have 2s animation